.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,76 @@
+# Copied from NumPy https://github.com/numpy/numpy/pull/25894
+# https://github.com/numpy/numpy/blob/d2d2c25fa81b47810f5cbd85ea6485eb3a3ffec3/.github/workflows/emscripten.yml
+#
+
+name: Test Emscripten/Pyodide build
+
+on:
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+  push:
+    branches:
+      - main
+      - maintenance/**
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: 3
```

Comment:
Just curious, at what tool is this variable aimed at?

Reply:
I have been referencing this from https://learn.scientific-python.org/development/guides/tasks/#nox through which GHA can display coloured outputs for tests – it works well for regular Python/`pytest` test suites. I don't think it's working in this Node.js test runner, however. I am happy with either removing it or keeping it for other jobs. :)